docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: "Configure OIDC"
+title: "Configure OIDC"
+description: "Learn how to configure OIDC so Palette displays the Spectro VM Dashboard."
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["vmo", "oidc"]
+---
+
+
+Palette displays the VM dashboard based on OpenID Connect (OIDC) Identity Provider options you enable in the Kubernetes layer of the cluster profile you will use for your Virtual Machine Orchestrator (VMO) cluster.
+
+
+## Prerequisites
+
+- A configured cluster profile. For more information, review [Create a Cluster Profile](/cluster-profiles/task-define-profile).
+
+
+## Configure OIDC Options
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** click **Profiles**.
+
+3. Select the cluster profile to update. Palette displays profile details and the profile stack.
+
+4. Select the Kubernetes layer in the profile stack, and choose **None** or **Palette** as the OIDC Identity Provider.
```

Comment:
Is `Custom`  or `Inherit from Tenant` not a value?

Reply:
@karl-cardenas-coding They are, but from what I understand you must choose either None or Palette in order to display the dashboard. That's why I asked about the one paragraph in the Add Roles and Role Bindings doc for VMO.

Reply:
Added all `Custom` and `Inherit from Tenant`.